### PR TITLE
Update to latest monaco-editor-wrapper and monaco-editor-react

### PR DESCRIPTION
**Pull request by kaisalmen**
Updates the user config / app init code and any dependent code. The whole init and utility code regarding `monaco-editor-wrapper` and `@typefox/monaco-editor-react` and its bundle generation has been moved to its own package: **langium-website-core**

Hugo code only uses that bundle which prevents any complex javascript issues when generating the site.

All showcases and the playground work again!

This PR now uses the final version `3.3.0` of `monaco-editor-wrapper` and `2.3.0` of `@typefox/monaco-editor-react`

### CODE REVIEW

File: foundation/package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        "dist/index"
+      ]
+    }
+  },
+  "files": [
+    "dist",
+    "bundle",
+    "src",
+    "LICENSE",
+    "README.md"
```
Comment (montymxb) [inline foundation/package.json:36]: Should add this README into the package

File: foundation/package.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "langium-website-foundation",
```
Comment (montymxb) [inline foundation/package.json:2]: Foundation sounds like an organization, core might be better here.

File: hugo/assets/scripts/arithmetics/arithmetics-tools.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { monaco } from "monaco-editor-wrapper/.";
+import { monaco } from "langium-website-foundation/bundle";
```
Comment (montymxb) [inline hugo/assets/scripts/arithmetics/arithmetics-tools.tsx:1]: langium-website-core/bundle might be good here.

File: hugo/package.json [hunk 4]
```diff
@@ -6,38 +6,34 @@
   "private": true,
   "scripts": {
     "clean:static": "shx rm -rf ./static/showcase ./static/playground ./static/libs",
-    "copy:libs:prepare": "shx mkdir -p ./static/libs/monaco-editor-workers/workers",
-    "copy:monaco-editor-wrapper": "npm run copy:libs:prepare && shx cp -fr ../node_modules/monaco-editor-wrapper/bundle ./static/libs/monaco-editor-wrapper",
+    "copy:libs:prepare": "shx mkdir -p ./static/libs/monaco-editor-workers/workers && shx mkdir -p ./static/libs/langium-website-foundation",
```
Comment (montymxb) [inline hugo/package.json:9]: We don't need to create a directory for the langium-website-foundation, nothing is copied there.

File: package.json [hunk 5]
```diff
@@ -5,22 +5,23 @@
...
-    "build": "npm run clean && npm run build --workspace=tailwind --workspace=hugo",
-    "clean": "shx rm -rf public && npm run clean --workspace=tailwind"
+    "watch": "npm run build --workspace foundation && concurrently -c gray,blue -k -n hugo,tailwind \"npm run watch --workspace hugo\" \"npm run watch --workspace tailwind\"",
+    "watch:gitpod": "npm run build --workspace foundation && concurrently -c gray,blue -k -n hugo,tailwind \"npm run watch:gitpod --workspace hugo\" \"npm run watch --workspace tailwind\"",
+    "build": "npm run clean && npm run build --workspace foundation --workspace=tailwind --workspace=hugo",
+    "clean": "shx rm -rf public && npm run clean --workspace foundation --workspace=tailwind"
   },
   "devDependencies": {
-    "concurrently": "~7.6.0",
+    "concurrently": "~8.2.1",
```
Comment (montymxb) [inline package.json:14]: Noting that concurrently v8.X.X dropped node12 support in favor of node18, which works just fine for us here.

**Review by montymxb — CHANGES_REQUESTED** [review]
- We should update the README to explain the purpose of the new package, and how it fits into the greater flow.
- Add a README to the new package itself, just to summarize it very briefly

Separately, I would consider **core** instead of **foundation** for the package name.

File: hugo/assets/scripts/arithmetics/arithmetics-tools.tsx (hunk 3 above)
Comment (kaisalmen) [reply hugo/assets/scripts/arithmetics/arithmetics-tools.tsx]: Changed to `core` in next commit ⬇️

File: hugo/package.json (hunk 4 above)
Comment (kaisalmen) [reply hugo/package.json]: Thanks for spotting that.

File: foundation/package.json (hunk 2 above)
Comment (kaisalmen) [reply foundation/package.json]: Changed to `core` in next commit ⬇️

File: package.json (hunk 5 above)
Comment (kaisalmen) [reply package.json:14]: Thanks.

**Comment by kaisalmen** [comment]
All things are addressed. `foundation` is renamed to `core`. README is added and a launch config for Chrome is there now, too.

**Review by montymxb — APPROVED** [review]
:octocat: approved! Changes look good, and verified locally.

**Pull request merged**